src/Client.php

```diff
@@ -75,7 +75,8 @@ public function post($url, $organizationId, array $data = [], array $params = []
         return $this->processResult($this->httpClient->post(
             $url,
             [
-                'query' => $this->getParams($organizationId, $data) + $params,
+                'query' => $params + $this->getParams($organizationId),
+                'form_params' => ['JSONString' => json_encode($data)],
```

Comment:
Just to be clear, `\GuzzleHttp\json_encode()` forces the 3rd argument (depth) for `\json_encode()` to be set (using `512` as default value). This behavior is fine for a string which is intended to be used in an URL (`getParams()`), but there is no sense in limiting the request body to an specific length too.